src/MidiPlayer.js

```diff
@@ -624,4 +625,36 @@ not.
         this.logging = logging;
         this.eventHandler.setLogger({ eventLogger, logging });
     }
+    /*
+     * get total duration time
+     * 2021 03 NormalUniversity
+     */
+    getDuration() {
+
+        return this.totalTime;
+    }
+    /**
+     * Seek Position.
+     * @param {any} [mSecond] milli seconds to be seek
+     * @example
+     *  let midiPlayer = new MidiPlayer({ logging: false, patchUrl: 'lib/pat/' });
+     *  midiPlayer.play({url: $url})
+     *  midiPlayer.seek({mSecond: $milliSeconds});
+     */
+    async seek({ mSecond}) {
+        if (this.totalTime>0) {
```

Comment:
It does not seem that `this.totalTime` is ever updated after it's been assigned a value of `0` on line 74. As a result, it looks like this condition will always be `false`. Did you test the code you wrote?